build.sc

```diff
@@ -84,6 +82,12 @@ object scalanet extends ScalaModule with PublishModule {
     }
   }
 
+  // Scoverage disabled
+  // object test extends ScoverageTests {
+  object test extends TestModule
+
+  object integrationTest extends TestModule
```

Comment:
Oooof, that's a long word to type on the command line. Perhaps it could be `it`?